MySQL CI: ValueError: unrecognize dtype 1 when using TestExecutorTrainAndPredictionDNNClassifierDENSE

Based on https://github.com/sql-machine-learning/sqlflow/pull/2970 .

```
2020-09-30T01:49:28.2810874Z [0m[0m=== RUN   TestExecutorTrainAndPredictionDNNClassifierDENSE
2020-09-30T01:49:41.8794912Z [0m[91m--- FAIL: TestExecutorTrainAndPredictionDNNClassifierDENSE (13.60s)
2020-09-30T01:49:41.8797465Z [0m[0m    executor_ir_test.go:242: 
2020-09-30T01:49:41.8798407Z [0m[0m        	Error Trace:	executor_ir_test.go:242
2020-09-30T01:49:41.8799757Z [0m[0m        	            				executor_ir_test.go:222
2020-09-30T01:49:41.8800813Z [0m[0m        	Error:      	Should be true
2020-09-30T01:49:41.8802777Z [0m[0m        	Test:       	TestExecutorTrainAndPredictionDNNClassifierDENSE
2020-09-30T01:49:41.8805000Z [0m[0m        	Messages:   	runSQLProgram error: exit status 1
2020-09-30T01:49:41.8806318Z [0m[0m        	            	Traceback (most recent call last):
2020-09-30T01:49:41.8807505Z [0m[0m        	            	  File "<stdin>", line 13, in <module>
2020-09-30T01:49:41.8809332Z [0m[0m        	            	  File "/home/sqlflow/actions-runner/_work/sqlflow/sqlflow/python/runtime/local/submitter.py", line 116, in submit_local_pred
2020-09-30T01:49:41.8811054Z [0m[0m        	            	    model=model)
2020-09-30T01:49:41.8813225Z [0m[0m        	            	  File "/home/sqlflow/actions-runner/_work/sqlflow/sqlflow/python/runtime/local/tensorflow_submitter/predict.py", line 88, in pred
2020-09-30T01:49:41.8815137Z [0m[0m        	            	    predict_generator, selected_cols)
2020-09-30T01:49:41.8816938Z [0m[0m        	            	  File "/home/sqlflow/actions-runner/_work/sqlflow/sqlflow/python/runtime/tensorflow/predict.py", line 229, in estimator_predict
2020-09-30T01:49:41.8818672Z [0m[0m        	            	    is_xgboost=False)
2020-09-30T01:49:41.8820184Z [0m[0m        	            	  File "/home/sqlflow/actions-runner/_work/sqlflow/sqlflow/python/runtime/db.py", line 209, in read_features_from_row
2020-09-30T01:49:41.8821684Z [0m[0m        	            	    feature_metas[name], name, is_xgboost)
2020-09-30T01:49:41.8823189Z [0m[0m        	            	  File "/home/sqlflow/actions-runner/_work/sqlflow/sqlflow/python/runtime/db.py", line 100, in read_feature
2020-09-30T01:49:41.8824391Z [0m[0m        	            	    feature_spec["dtype"]))
2020-09-30T01:49:41.8825336Z [0m[0m        	            	ValueError: unrecognize dtype 1
2020-09-30T01:49:41.8826392Z [0m[0m        	            	: Start predicting using estimator model...
```

--- a/python/runtime/local/tensorflow_submitter/predict.py
+++ b/python/runtime/local/tensorflow_submitter/predict.py
@@ -53,7 +53,8 @@ def pred(datasource, select, result_table, pred_label_name, model):
 
     field_descs = get_ordered_field_descs(train_fc_map)
     feature_column_names = [fd.name for fd in field_descs]
-    feature_metas = dict([(fd.name, fd.to_dict()) for fd in field_descs])
+    feature_metas = dict([(fd.name, fd.to_dict(dtype_to_string=True))
+                          for fd in field_descs])
     feature_columns = compile_ir_feature_columns(train_fc_map,
                                                  model.get_type())
 

--- a/python/runtime/local/xgboost_submitter/evaluate.py
+++ b/python/runtime/local/xgboost_submitter/evaluate.py
@@ -92,7 +92,8 @@ def evaluate(datasource,
 
     field_descs = get_ordered_field_descs(train_fc_map)
     feature_column_names = [fd.name for fd in field_descs]
-    feature_metas = dict([(fd.name, fd.to_dict()) for fd in field_descs])
+    feature_metas = dict([(fd.name, fd.to_dict(dtype_to_string=True))
+                          for fd in field_descs])
 
     # NOTE: in the current implementation, we are generating a transform_fn
     # from the COLUMN clause. The transform_fn is executed during the process
@@ -111,15 +112,16 @@ def evaluate(datasource,
     with temp_file.TemporaryDirectory() as tmp_dir_name:
         pred_fn = os.path.join(tmp_dir_name, "predict.txt")
 
-        dpred = xgb_dataset(datasource=datasource,
-                            fn=pred_fn,
-                            dataset_sql=select,
-                            feature_metas=feature_metas,
-                            feature_column_names=feature_column_names,
-                            label_meta=train_label_desc.to_dict(),
-                            cache=True,
-                            batch_size=10000,
-                            transform_fn=transform_fn)
+        dpred = xgb_dataset(
+            datasource=datasource,
+            fn=pred_fn,
+            dataset_sql=select,
+            feature_metas=feature_metas,
+            feature_column_names=feature_column_names,
+            label_meta=train_label_desc.to_dict(dtype_to_string=True),
+            cache=True,
+            batch_size=10000,
+            transform_fn=transform_fn)
 
         for i, pred_dmatrix in enumerate(dpred):
             feature_file_name = pred_fn + "_%d" % i

--- a/python/runtime/local/xgboost_submitter/explain.py
+++ b/python/runtime/local/xgboost_submitter/explain.py
@@ -181,11 +181,13 @@ def xgb_native_explain(booster, datasource, result_table):
 def shap_explain(booster, datasource, select, summary_params, result_table,
                  model):
     train_fc_map = model.get_meta("features")
-    label_meta = model.get_meta("label").get_field_desc()[0].to_dict()
+    label_meta = model.get_meta("label").get_field_desc()[0].to_dict(
+        dtype_to_string=True)
 
     field_descs = get_ordered_field_descs(train_fc_map)
     feature_column_names = [fd.name for fd in field_descs]
-    feature_metas = dict([(fd.name, fd.to_dict()) for fd in field_descs])
+    feature_metas = dict([(fd.name, fd.to_dict(dtype_to_string=True))
+                          for fd in field_descs])
 
     # NOTE: in the current implementation, we are generating a transform_fn
     # from the COLUMN clause. The transform_fn is executed during the process

--- a/python/runtime/local/xgboost_submitter/predict.py
+++ b/python/runtime/local/xgboost_submitter/predict.py
@@ -52,7 +52,8 @@ def pred(datasource, select, result_table, pred_label_name, model):
 
     field_descs = get_ordered_field_descs(train_fc_map)
     feature_column_names = [fd.name for fd in field_descs]
-    feature_metas = dict([(fd.name, fd.to_dict()) for fd in field_descs])
+    feature_metas = dict([(fd.name, fd.to_dict(dtype_to_string=True))
+                          for fd in field_descs])
 
     # NOTE: in the current implementation, we are generating a transform_fn
     # from the COLUMN clause. The transform_fn is executed during the process

--- a/python/runtime/local/xgboost_submitter/train.py
+++ b/python/runtime/local/xgboost_submitter/train.py
@@ -76,8 +76,9 @@ def train(original_sql,
     feature_column_list = fc_map["feature_columns"]
     field_descs = get_ordered_field_descs(fc_map_ir)
     feature_column_names = [fd.name for fd in field_descs]
-    feature_metas = dict([(fd.name, fd.to_dict()) for fd in field_descs])
-    label_meta = label_column.get_field_desc()[0].to_dict()
+    feature_metas = dict([(fd.name, fd.to_dict(dtype_to_string=True))
+                          for fd in field_descs])
+    label_meta = label_column.get_field_desc()[0].to_dict(dtype_to_string=True)
 
     # NOTE: in the current implementation, we are generating a transform_fn
     # from the COLUMN clause. The transform_fn is executed during the process

